libs/scalalib/src/mill/scalalib/CrossSbtPlatformModule.scala

```diff
@@ -0,0 +1,21 @@
+package mill.scalalib
+
+/**
+ * A [[SbtPlatformModule]] suited to be used with [[mill.api.Cross]]. It supports additional source
+ * directories with the scala version pattern as suffix (src/main/scala-{scalaversionprefix}), e.g.
+ *  - src/main/scala-2
+ *  - src/main/scala-2.12
+ *  - src/main/scala-2.13
```

Comment:
How does the `-2`, `-2.12`, `-2.13` suffixes interact with the `-jvm`, `-js`, and `-native` suffixes? Can they be used together? I assume so, but we should mention how it works in the doccoment

Reply:
Updated Scaladocs.